pages/common/eva.md

```diff
@@ -0,0 +1,20 @@
+# eva
+
+> Simple calculator REPL, similar to `bc`, with syntax highlighting and persistent history.
+> More information <https://github.com/NerdyPepper/eva>.
+
+- Run calculator in interactive mode:
+
+`eva`
+
+- Calculate the result of an expression:
+
+`eva <<< "{{(1 + 2) * 2 ^ 2}}"`
```

Comment:
I see from the linked homepage (https://github.com/NerdyPepper/eva) that this could be done by simply supplying the expression as first argument. Like:

```suggestion
`eva "{{(1 + 2) * 2 ^ 2}}"`
```

This and the following examples should be modified accordingly, there is no need to use the bash redirection `<<<`.